.github/workflows/quickstart_ci.yml

```diff
@@ -89,6 +89,10 @@ jobs:
           cd ${{ inputs.QUICKSTART_PATH }}
           echo "Building provisioned server..."
           mvn -U -B -fae clean package -Pprovisioned-server
+          echo "Add quickstartUser..."
```

Comment:
this changes the abstract workflow to always add the quickstart users (roles are the ones I found on different quickstarts)